### PR TITLE
Replace clean-tag with custom util

### PR DESCRIPTION
**Pull request by evadb**
This PR replaces clean-tag and clean-element with a custom util cleanProps, and emotion's built in shouldForwardProp util. Basically, the cleanProp util compares the incoming prop to the list of styled system props, and returns true to shouldForwardProp if it is not in the list. A prop that is forwarded will appear in the rendered HTML. The goal of this util is to make sure no styled-system props appear in the final render, as they are only useful for generating the CSS.

### CODE REVIEW

**Comment by evadb** [comment]
@sleepycat hence the WIP! :) Took a bit to get the tests right without suffering, but next step is to replace clean-tag everywhere with this and then remove clean tag

File: frontend/locale/fr/messages.json [hunk 1]
```diff
@@ -27,4 +27,4 @@
   "a website": "d’un site web",
   "an email address": "une adresse courriel",
   "loading...": "chargement en cours..."
-}
\ No newline at end of file
+}
```
Comment (sastels) [inline frontend/locale/fr/messages.json]: good catch

File: frontend/src/utils/cleanProps.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+export const cleanProps = prop => {
+  if (!styledSystemProps.includes(prop) && prop !== 'as') return true
+  return false
+}
+
+export const styledSystemProps = [
```
Comment (sastels) [inline frontend/src/utils/cleanProps.js]: We'll have to remember to update this when `styled-system` changes.

Comment (evadb) [reply frontend/src/utils/cleanProps.js]: the good news is the actual list of props is very unlikely to change

**Pull request merged**